lib/topic_query.rb

```diff
@@ -19,6 +19,8 @@ def self.validators
         array_or_string = lambda { |x| Array === x || String === x }
 
         {
+          before: zero_up_to_max_int,
+          bumped_before: zero_up_to_max_int,
```

Comment:
This validator chosen based on these lines:

https://github.com/discourse/discourse/blob/main/lib/topic_query.rb#L834-L835

Negative numbers are not respected, so might as well validate that.